addon/components/bs4/bs-form/element/help-text.js

```diff
@@ -0,0 +1,5 @@
+import FormElementHelpText from 'ember-bootstrap/components/base/bs-form/element/help-text';
+
+export default FormElementHelpText.extend({
+  classNames: ['form-text']
```

Comment:
I thought this was elsewhere in the validation feedback. I found it [here](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/base/bs-form/element.js#L98) in the docs, but I don't know what happened to the code reference to it. It seems this class is used for any feedback presented in the form.